contrib/demo-rgb.sh

```diff
@@ -158,3 +158,34 @@ btc-cold finalize --publish testnet ${PSBT}
 
 # Now, once the transaction will be mined, the verification should pass
 rgb consignment validate ${CONSIGNMENT}
+
+# -- CONSUME AND UNLOCK ASSET ------------------------------------------------
+# Go to a remote server / other machine and do the following
+
+# First, we must get all the information required to consume the consignment file. 
```

Comment:
could you please remove spaces at end of lines? I've noticed it in this and other 4 lines (I suggest highlighting trailing whitespace in your editor or setting `git config --global diff.wsErrorHighlight all`)

Reply:
WOW, thanks. This trick is usefully